### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by t007rushi**
## Header 
 - title + logo
 - options
 - search bar
 - profile logo
## Image Slider carousel
- option to jump and prev/next

recording

<details><summary>Browser metadata</summary>

```
Path:      /
Browser:   Chrome 100.0.4896.75 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1--gamsey.netlify.app%2F&os=Windows&os_version=10&browser=Chrome&browser_version=100.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-1--gamsey.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6251e2170358af0008b7e38e&commentID=6251e4403799d45fc6d32ade&resolution=resolved)

### CODE REVIEW

**Review by narharikale — APPROVED** [review]
Great work rushi 👌
Clean and concise code 
Loved the implementation of slider 🤩

File: src/frontend/Layout/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <img
+          className="web-logo-img"
+          src="./assets/gamseylogo.jpg"
+          alt="logo"
+        />
+        <h1 className="flex-row web-logo">Gamsey</h1>
+        <div className="flex-row gap-btwn header-options">
+          {optionsData.map((option) => {
+            return (
+              <div className="flex-col center-it">
```
Comment (JantuDeb) [inline src/frontend/Layout/Header/Header.jsx]: add a `key` attribute

File: src/frontend/components/ImgSlider/ImgSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import Slider from "react-slick";
+import "./slider.css";
+
+export const ImgSlider = () => {
+  var settings = {
```
Comment (JantuDeb) [inline src/frontend/components/ImgSlider/ImgSlider.jsx]:
```js
const settings ={...}

```

**Review by JantuDeb — APPROVED** [review]
Code looks clean and structured

**Pull request merged**